docs/source/eFeatures.rst

```diff
@@ -724,6 +724,21 @@ a given number of ms (default: 5) after the peak
 - **Required features**: LibV1:peak_indices
 - **Units**: mV
 
+
+LibV1 : AHP_depth_slow
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
```

Comment:
Could you put only one empty line here instead of two?

Reply:
voila